### DISCUSSION TITLE
Improve labels for issues

### DISCUSSION THREAD

**Discussion by reneeb** [RotherOSS/otobo#1536] [General]
I would propose to use more labels:

* frontend
* docker
* generic_interface
* processmangement
* ...

This woule make issue triage easier...

**Comment by bschmalhofer** [comment]
Sounds good to me.

**Comment by svenoe** [comment]
Totally fine with me, too. Is one other classification layer enough? (Would you consider frontend and process management being on the same one?) Or rather two, like "process management; frontend; bug;" for an erroneously shown label in the processmanagement, e.g.? I don't have strong opinions here. We can just try out what proves useful and what doesn't...

Any more detailed wishes? :)

**Reply by bschmalhofer** [reply]
I would not nest the tags. In the above mentioned case I'm totally fine with three independent tags: 'process management', 'frontend', 'bug'
